### DISCUSSION TITLE
Consider allowing users to simply specific SRA ID for a run

### DISCUSSION THREAD

**Discussion by abhi18av** [TORCH-Consortium/MAGMA#66] [Ideas]
We should accomodate the use-case where people only have SRA IDs.

As per the discussion with Lennert, we can use some default values in this case.

**Comment by TimHHH** [comment]
Sure, do I understand this right and that XBS-nf would automatically download these files from SRA upon specifying their SRA ID's in the sample file? That would be cool.

**Reply by abhi18av** [reply]
Yup, Nextflow natively supports this.

Or we can basically free-ride upon the https://nf-co.re/fetchngs pipeline, educate the user in the docs and separate the concerns. 

Thoughts?

**Reply by TimHHH** [reply]
We best go with whatever is easiest/quickest to implement. This is a nice feature, but not highest priority.

**Reply by abhi18av** [reply]
Agreed - for the time being, I also think that its best if we keep the core small.
